src/main/java/betterquesting/api/utils/JsonHelper.java

```diff
@@ -177,7 +177,7 @@ public static void ClearCompoundTag(NBTTagCompound tag)
 	public static JsonObject ReadFromFile(File file)
 	{
 		Future<JsonObject> task = BQThreadedIO.INSTANCE.enqueue(() -> {
-			if(file == null || !file.exists())
+			if(file == null || !file.exists() || file.getName().contains(".DS_Store") || file.getName().contains("malformed_"))
```

Comment:
we could generalize this to a config or something but it covers the main one (stupid .DS_Store) and if something else occurs, it'll only ever generate one per start.